docs/cpu_architecture.md

```diff
@@ -26,32 +26,36 @@ spec:
         ...
 ```
 
-## Overriding Architecture for Individual Nodes
+If no value is provided, the SiteConfig operator sets the default `x86_64` value.
+
+## Overriding architecture on node level
 
-Each node in the `nodes` list can specify its `cpuArchitecture` field to override the default set at the cluster level.
+For each node in the `nodes` list, you can specify the node's `cpuArchitecture` field to override the cluster-level value. If no value is provided, the node inherits the cluster-level value, if specified. Otherwise, it defaults to the `x86_64` value.
 
-If no value is provided, the node inherits the cluster-level default (if specified). Otherwise, it defaults to `x86_64`.
+> **Important:** When deploying single-node OpenShift clusters with the Image Based Installer Operator, you must override the default CPU architecture for individual nodes.
```

Comment:
IBI does not work with anything besides x86 currently, so I'm not sure why this needs to be called out for SNO specifically?

Reply:
>We don't mention that IBI only works with x86. Should we?
Its at the bottom of the file actually, here: https://github.com/stolostron/siteconfig/pull/251/files#diff-44c02513fbc5d31a9e67e51bc34937dde6420937c46160260ff1197e3a35713cR61

Otherwise LGTM, will let Sharat approve

Reply:
/cc @sakhoury

Reply:
/lgtm

Reply:
Thanks, I see it now. However, I have a follow-up question/suggestion. If we leave the docs as they are now for [this part](https://github.com/stolostron/siteconfig/pull/251/files#diff-44c02513fbc5d31a9e67e51bc34937dde6420937c46160260ff1197e3a35713cR35-R43). It's a bit confusing as the example for overriding the CPU architecture, which is required when using SNOs installed with IBIO, shows an ARM configuration, which is not supported. I think we should either change the architecture to `x86_64` or have dedicated section for SNOs installed with IBIO

Reply:
Hmm it does feel a bit misleading. I think updating the example to use x86 is fine its just to show how to set the field

Reply:
Thanks, I updated it. I also removed the "Configuring ARM nodes for Image Based Installer" section as that can be misleading too. Instead, I added a note at the top that it's not support with IBI: https://github.com/stolostron/siteconfig/pull/251/commits/f3ba3c976a99b4b79408c316e0d4d5323cff4b61